bsp/esp32_p4_function_ev_board/esp32_p4_function_ev_board.c

```diff
@@ -38,36 +38,30 @@ static lv_indev_t *disp_indev = NULL;
 #endif // (BSP_CONFIG_NO_GRAPHIC_LIB == 0)
 
 sdmmc_card_t *bsp_sdcard = NULL;    // Global uSD card handler
-static bool i2c_initialized = false;
 static TaskHandle_t usb_host_task;  // USB Host Library task
+i2c_master_bus_handle_t bsp_i2c_handle = NULL;   // Global i2c master bus handler
```

Comment:
Please, can we add get function like this: https://github.com/espressif/esp-bsp/blob/3284337601d3e65d0a60d4a1c4428b4e581e4c67/bsp/esp-box/esp-box.c#L116

and this: https://github.com/espressif/esp-bsp/blob/3284337601d3e65d0a60d4a1c4428b4e581e4c67/bsp/esp-box/include/bsp/esp-box.h#L241

Reply:
It's ok.

Reply:
@100312dog please, do you plan add mentioned functions?